app/views/state_file/questions/nj_tenant_rent_paid/edit.html.erb

```diff
@@ -7,6 +7,7 @@
   <h1 class="h2"><%= t(".title", filing_year: current_tax_year) %></h1>
 
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <p><%= t('.instructions_html') %></p>
```

Comment:
[pebble] This will nest a `<p>` within a `<p>` because the content also contains a p. It should only be one or the other